Makefile

```diff
@@ -22,7 +22,7 @@ serve:
 run:
 	pipenv run inv run
 
-test: flake8 unittests
```

Comment:
The flake8 issues don't seem that bad. I think it's just worth fixing them on this while you're here

Reply:
It was mostly the vulture failures which needed a bit more investigation, so wondered if we could just fix them altogether

Reply:
Christ that's a lot of warnings from vulture. I was just thinking it'd be good to have flake8 running always. Defo need a card on the vulture stuff